src/json-api-koa.ts

```diff
@@ -0,0 +1,50 @@
+import { Context } from "koa";
+
+import Application from "./application";
+import { JsonApiDocument, Operation, OperationResponse } from "./types";
+
+export default function jsonApiKoa(app: Application) {
+  const URL_REGEX = new RegExp(app.types.map(t => t.name).join("|"), "i");
+
+  return async (ctx: Context, next: Function) => {
+    if (ctx.url.match(URL_REGEX) && app.types.length) {
+      const ops: Operation[] = convertHttpRequestToOperations(ctx);
+      const result: OperationResponse[] = await app.executeOperations(ops);
+      ctx.body = convertOperationsResponsesToHttpResponse(ctx, result);
+    } else if (ctx.url.match("/bulk")) {
+      return {
```

Comment:
@Exelord this should be `ctx.body`

Reply:
`ctx.body` is an alias for `ctx.reponse.body`. Here we want to use the `ctx.request.body` - the incoming data